### PR TITLE
Add new trigger type endpoint to processes

### PR DESCRIPTION
**Pull request by NicolasMahe**
related to https://github.com/liteflow-labs/liteflow-js/issues/111

- Add new trigger type endpoint to processes: by setting the key `endpointKey`
- Update dev command to start a local HTTP gateway service
- Update deploy command to create the endpoint on the platform and the blockchain

### CODE REVIEW

File: packages/cli/src/commands/deploy.ts [hunk 1]
```diff
@@ -181,7 +194,19 @@ export default class Deploy extends Command {
...
+    },
+    {
+      title: 'Creating webhooks',
+      skip: () => Object.keys(this.webhooksToDeploy).length === 0,
+      task: () => new Listr(Object.keys(this.webhooksToDeploy).map(webhookKey => ({
+        title: webhookKey,
+        task: async (ctx, task) => {
+          const deployment = await deploy.webhook({ webhookKey }, userId, projectId)
+          deployment.on('data', x => task.output = x.message)
+          const webhook = await deployment.promise()
```
Comment (antho1404) [inline packages/cli/src/commands/deploy.ts:209]: ;)
```suggestion
          await deployment.promise()
```

File: packages/cli/src/commands/deploy.ts [hunk 2]
```diff
@@ -181,7 +194,19 @@ export default class Deploy extends Command {
...
+        }
+      })))
+    },
+    {
+      title: 'Creating webhooks',
+      skip: () => Object.keys(this.webhooksToDeploy).length === 0,
+      task: () => new Listr(Object.keys(this.webhooksToDeploy).map(webhookKey => ({
+        title: webhookKey,
+        task: async (ctx, task) => {
+          const deployment = await deploy.webhook({ webhookKey }, userId, projectId)
```
Comment (antho1404) [inline packages/cli/src/commands/deploy.ts:207]: To keep it consistent and also this way we have the webhookHash in the db
```suggestion
          const deployment = await deploy.webhook(this.webhooksToDeploy[webhookKey], userId, projectId)
```

Comment (NicolasMahe) [reply packages/cli/src/commands/deploy.ts]: the hash is already calculated on the cloud function and saved ;)

File: packages/cli/src/commands/deploy.ts (hunk 1 above)
Comment (NicolasMahe) [reply packages/cli/src/commands/deploy.ts]: done

**Pull request merged**